app/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,86 @@
+        <item name="android:listDivider">@color/rsb_list_divider</item>
+
+        <item name="toolbarTheme">@style/Theme.MP.Toolbar</item>
+        <item name="passcodeTheme">@style/Theme.AppCompat.Light.NoActionBar</item>
+    </style>
+
+    <style name="Theme.MP.Toolbar">
+        <item name="colorControlNormal">@android:color/white</item>
+        <item name="toolbarStyle">@style/Widget.MP.Toolbar</item>
+    </style>
+
+    <style name="Theme.MP.Survey">
+        <item name="toolbarTheme">@style/Theme.MP.Toolbar.Main</item>
+        <item name="colorButtonNormal">@color/sageResearchPrimary</item>
+        <item name="colorPrimaryDark">@color/sageResearchPrimaryDark</item>
+        <item name="android:windowContentOverlay">@null</item>
+    </style>
+
+    <style name="Theme.MP.Toolbar.Transparent">
+        <item name="colorControlNormal">@color/sageResearchPrimary</item>
+        <item name="colorPrimary">@color/sageResearchPrimary</item>
+        <item name="colorAccent">@color/sageResearchPrimaryLight</item>
+        <item name="toolbarStyle">@style/Widget.BP.Toolbar.Transparent</item>
+    </style>
+
+    <style name="Theme.MP.Main">
+        <item name="toolbarTheme">@style/Theme.MP.Toolbar.Main</item>
+        <item name="colorPrimaryDark">@color/sageResearchPrimaryDark</item>
+    </style>
+
+    <style name="Theme.MP.Toolbar.Main">
+        <item name="toolbarStyle">@style/Base.Widget.MP.Toolbar</item>
+    </style>
+
+    <style name="Theme.MP.Dialog" parent="Theme.AppCompat.Light.Dialog.Alert">
+        <item name="colorAccent">@color/colorAccent</item>
+        <item name="android:textColorPrimary">@color/sageResearchSecondary</item>
+        <item name="android:textColor">@color/sageResearchSecondary</item>
+        <item name="android:background">@android:color/white</item>
+        <item name="android:windowIsFloating">true</item>
+        <item name="android:windowAnimationStyle">@android:style/Animation.Dialog</item>
+    </style>
+
+    <style name="MP.ButterScotch.RoundedCornerMaterialButton" parent="Widget.AppCompat.Button.Colored">
+        <item name="android:background">@drawable/mp_rounded_button_butterscotch</item>
+        <item name="android:textColor">@color/appTextDark</item>
+        <item name="android:textAllCaps">false</item>
+        <item name="android:fontFamily">@font/lato_bold</item>
+        <item name="android:textSize">19sp</item>
+    </style>
+
+    <style name="MP.Gray.RoundedCornerMaterialButton" parent="Widget.AppCompat.Button.Colored">
+        <item name="android:background">@drawable/mp_rounded_button_gray</item>
+        <item name="android:textColor">@color/sageResearchSecondary</item>
+        <item name="android:textAllCaps">false</item>
+        <item name="android:fontFamily">@font/lato_bold</item>
+        <item name="android:textSize">19sp</item>
+    </style>
+
+    <style name="BP.SkipButton" parent="Widget.AppCompat.Button.Borderless">
```

Comment:
This is great for any button type that is just an image.  It adds the ripple effect on android < 21 and still has a click affect on OS lower than 21.